src/stylesheets/core/placeholders/_table.scss

```diff
@@ -49,10 +46,114 @@
     border-left: 0;
     border-right: 0;
   }
+}
+
+%usa-table--compact {
+  th,
+  td {
+    padding: units(0.5) units(1.5);
+  }
+}
+
+%usa-table--striped {
+  tbody {
+    tr:nth-child(odd) {
+      td,
+      th {
+        background-color: color($theme-table-stripe-background-color);
+      }
+    }
+  }
+}
+
+@mixin table-stacked-styles {
+  thead {
+    display: none;
+  }
+
+  th,
+  td {
+    display: block;
+    width: 100%;
+    border-bottom-width: 0;
+  }
+
+  tr {
+    @include u-border-bottom(0.5, $theme-table-border-color);
+    display: block;
+    width: 100%;
+    border-top-width: 0;
+    th:first-child,
+    td:first-child {
+      border-top-width: 0;
+    }
+    &:nth-child(odd) {
+      td,
+      th {
+        background-color: inherit;
+      }
+    }
 
-  th {
     &:first-child {
-      padding-left: 0;
+      th:first-child,
+      td:first-child {
+        @include u-border-top(0.5, $theme-table-border-color);
+      }
     }
   }
+
+  th[data-label],
+  td[data-label] {
+    padding-bottom: units(2.5);
+
+    &:before {
+      content: attr(data-label);
+      display: block;
+      font-weight: $theme-font-weight-bold;
```

Comment:
Should we keep use of`font-weight: fw("bold")` vs `$theme-font-weight-bold` consistent?

Reply:
Fixed!